File: PST/FUEL Source Code/FUEL/Backup/My/Resources/Resources.cs
// Decompiled with JetBrains decompiler
// Type: FUEL.My.Resources.Resources
// Assembly: FUEL, Version=2.4.8.0, Culture=neutral, PublicKeyToken=null
// MVID: A3D630D0-0620-425D-8CFB-B67B2AC75507
// Assembly location: C:\hp\FUEL\FUEL.dll

using Microsoft.VisualBasic;
using Microsoft.VisualBasic.CompilerServices;
using System.CodeDom.Compiler;
using System.ComponentModel;
using System.Diagnostics;
using System.Drawing;
using System.Globalization;
using System.Resources;
using System.Runtime.CompilerServices;

namespace FUEL.My.Resources
{
  [DebuggerNonUserCode]
  [StandardModule]
  [GeneratedCode("System.Resources.Tools.StronglyTypedResourceBuilder", "4.0.0.0")]
  [CompilerGenerated]
  [HideModuleName]
  internal sealed class Resources
  {
    private static ResourceManager resourceMan;
    private static CultureInfo resourceCulture;

    [EditorBrowsable(EditorBrowsableState.Advanced)]
    internal static ResourceManager ResourceManager
    {
      get
      {
        if (object.ReferenceEquals((object) FUEL.My.Resources.Resources.resourceMan, (object) null))
          FUEL.My.Resources.Resources.resourceMan = new ResourceManager("FUEL.Resources", typeof (FUEL.My.Resources.Resources).Assembly);
        return FUEL.My.Resources.Resources.resourceMan;
      }
    }

    [EditorBrowsable(EditorBrowsableState.Advanced)]
    internal static CultureInfo Culture
    {
      get => FUEL.My.Resources.Resources.resourceCulture;
      set => FUEL.My.Resources.Resources.resourceCulture = value;
    }

    internal static Bitmap Bad_PST_Box => (Bitmap) RuntimeHelpers.GetObjectValue(FUEL.My.Resources.Resources.ResourceManager.GetObject(nameof (Bad_PST_Box), FUEL.My.Resources.Resources.resourceCulture));

    internal static Bitmap Error_icon => (Bitmap) RuntimeHelpers.GetObjectValue(FUEL.My.Resources.Resources.ResourceManager.GetObject(nameof (Error_icon), FUEL.My.Resources.Resources.resourceCulture));

    internal static Bitmap Error_icon_sm => (Bitmap) RuntimeHelpers.GetObjectValue(FUEL.My.Resources.Resources.ResourceManager.GetObject(nameof (Error_icon_sm), FUEL.My.Resources.Resources.resourceCulture));

    internal static Bitmap FluctuatingPressure => (Bitmap) RuntimeHelpers.GetObjectValue(FUEL.My.Resources.Resources.ResourceManager.GetObject(nameof (FluctuatingPressure), FUEL.My.Resources.Resources.resourceCulture));

    internal static Bitmap frown_sm => (Bitmap) RuntimeHelpers.GetObjectValue(FUEL.My.Resources.Resources.ResourceManager.GetObject(nameof (frown_sm), FUEL.My.Resources.Resources.resourceCulture));

    internal static Bitmap Good_or_Tick_icon => (Bitmap) RuntimeHelpers.GetObjectValue(FUEL.My.Resources.Resources.ResourceManager.GetObject(nameof (Good_or_Tick_icon), FUEL.My.Resources.Resources.resourceCulture));

    internal static Bitmap Good_or_Tick_icon_sm => (Bitmap) RuntimeHelpers.GetObjectValue(FUEL.My.Resources.Resources.ResourceManager.GetObject(nameof (Good_or_Tick_icon_sm), FUEL.My.Resources.Resources.resourceCulture));

    internal static Bitmap happy_sm => (Bitmap) RuntimeHelpers.GetObjectValue(FUEL.My.Resources.Resources.ResourceManager.GetObject(nameof (happy_sm), FUEL.My.Resources.Resources.resourceCulture));

    internal static Bitmap idk1 => (Bitmap) RuntimeHelpers.GetObjectValue(FUEL.My.Resources.Resources.ResourceManager.GetObject(nameof (idk1), FUEL.My.Resources.Resources.resourceCulture));

    internal static Bitmap idk2 => (Bitmap) RuntimeHelpers.GetObjectValue(FUEL.My.Resources.Resources.ResourceManager.GetObject(nameof (idk2), FUEL.My.Resources.Resources.resourceCulture));

    internal static Bitmap Kinked_Tube_possible => (Bitmap) RuntimeHelpers.GetObjectValue(FUEL.My.Resources.Resources.ResourceManager.GetObject(nameof (Kinked_Tube_possible), FUEL.My.Resources.Resources.resourceCulture));

    internal static Bitmap Leak => (Bitmap) RuntimeHelpers.GetObjectValue(FUEL.My.Resources.Resources.ResourceManager.GetObject(nameof (Leak), FUEL.My.Resources.Resources.resourceCulture));

    internal static Bitmap NoPressure => (Bitmap) RuntimeHelpers.GetObjectValue(FUEL.My.Resources.Resources.ResourceManager.GetObject(nameof (NoPressure), FUEL.My.Resources.Resources.resourceCulture));

    internal static Bitmap oh_noes => (Bitmap) RuntimeHelpers.GetObjectValue(FUEL.My.Resources.Resources.ResourceManager.GetObject("oh noes", FUEL.My.Resources.Resources.resourceCulture));

    internal static Bitmap PinchedVentTube => (Bitmap) RuntimeHelpers.GetObjectValue(FUEL.My.Resources.Resources.ResourceManager.GetObject(nameof (PinchedVentTube), FUEL.My.Resources.Resources.resourceCulture));

    internal static Bitmap VerySmallVentDeltaP => (Bitmap) RuntimeHelpers.GetObjectValue(FUEL.My.Resources.Resources.ResourceManager.GetObject(nameof (VerySmallVentDeltaP), FUEL.My.Resources.Resources.resourceCulture));

    internal static Bitmap warning_sm => (Bitmap) RuntimeHelpers.GetObjectValue(FUEL.My.Resources.Resources.ResourceManager.GetObject(nameof (warning_sm), FUEL.My.Resources.Resources.resourceCulture));
  }
}

File: PST/FUEL/src/FUEL/My/Resources/Resources.vb
Imports Microsoft.VisualBasic
Imports Microsoft.VisualBasic.CompilerServices
Imports System
Imports System.CodeDom.Compiler
Imports System.ComponentModel
Imports System.Diagnostics
Imports System.Drawing
Imports System.Globalization
Imports System.Resources
Imports System.Runtime.CompilerServices

Namespace FUEL.My.Resources
    <DebuggerNonUserCode, StandardModule, GeneratedCode("System.Resources.Tools.StronglyTypedResourceBuilder", "4.0.0.0"), CompilerGenerated, HideModuleName> _
    Friend NotInheritable Class Resources
        ' Properties
        <EditorBrowsable(EditorBrowsableState.Advanced)> _
        Friend Shared ReadOnly Property ResourceManager As ResourceManager
            Get
                If Object.ReferenceEquals(Resources.resourceMan, Nothing) Then
                    Resources.resourceMan = New ResourceManager("FUEL.Resources", GetType(Resources).Assembly)
                End If
                Return Resources.resourceMan
            End Get
        End Property

        <EditorBrowsable(EditorBrowsableState.Advanced)> _
        Friend Shared Property Culture As CultureInfo
            Get
                Return Resources.resourceCulture
            End Get
            Set(ByVal Value As CultureInfo)
                Resources.resourceCulture = Value
            End Set
        End Property

        Friend Shared ReadOnly Property Bad_PST_Box As Bitmap
            Get
                Return DirectCast(Resources.ResourceManager.GetObject("Bad_PST_Box", Resources.resourceCulture), Bitmap)
            End Get
        End Property

        Friend Shared ReadOnly Property Error_icon As Bitmap
            Get
                Return DirectCast(Resources.ResourceManager.GetObject("Error_icon", Resources.resourceCulture), Bitmap)
            End Get
        End Property

        Friend Shared ReadOnly Property Error_icon_sm As Bitmap
            Get
                Return DirectCast(Resources.ResourceManager.GetObject("Error_icon_sm", Resources.resourceCulture), Bitmap)
            End Get
        End Property

        Friend Shared ReadOnly Property FluctuatingPressure As Bitmap
            Get
                Return DirectCast(Resources.ResourceManager.GetObject("FluctuatingPressure", Resources.resourceCulture), Bitmap)
            End Get
        End Property

        Friend Shared ReadOnly Property frown_sm As Bitmap
            Get
                Return DirectCast(Resources.ResourceManager.GetObject("frown_sm", Resources.resourceCulture), Bitmap)
            End Get
        End Property

        Friend Shared ReadOnly Property Good_or_Tick_icon As Bitmap
            Get
                Return DirectCast(Resources.ResourceManager.GetObject("Good_or_Tick_icon", Resources.resourceCulture), Bitmap)
            End Get
        End Property

        Friend Shared ReadOnly Property Good_or_Tick_icon_sm As Bitmap
            Get
                Return DirectCast(Resources.ResourceManager.GetObject("Good_or_Tick_icon_sm", Resources.resourceCulture), Bitmap)
            End Get
        End Property

        Friend Shared ReadOnly Property happy_sm As Bitmap
            Get
                Return DirectCast(Resources.ResourceManager.GetObject("happy_sm", Resources.resourceCulture), Bitmap)
            End Get
        End Property

        Friend Shared ReadOnly Property idk1 As Bitmap
            Get
                Return DirectCast(Resources.ResourceManager.GetObject("idk1", Resources.resourceCulture), Bitmap)
            End Get
        End Property

        Friend Shared ReadOnly Property idk2 As Bitmap
            Get
                Return DirectCast(Resources.ResourceManager.GetObject("idk2", Resources.resourceCulture), Bitmap)
            End Get
        End Property

        Friend Shared ReadOnly Property Kinked_Tube_possible As Bitmap
            Get
                Return DirectCast(Resources.ResourceManager.GetObject("Kinked_Tube_possible", Resources.resourceCulture), Bitmap)
            End Get
        End Property

        Friend Shared ReadOnly Property Leak As Bitmap
            Get
                Return DirectCast(Resources.ResourceManager.GetObject("Leak", Resources.resourceCulture), Bitmap)
            End Get
        End Property

        Friend Shared ReadOnly Property NoPressure As Bitmap
            Get
                Return DirectCast(Resources.ResourceManager.GetObject("NoPressure", Resources.resourceCulture), Bitmap)
            End Get
        End Property

        Friend Shared ReadOnly Property oh_noes As Bitmap
            Get
                Return DirectCast(Resources.ResourceManager.GetObject("oh noes", Resources.resourceCulture), Bitmap)
            End Get
        End Property

        Friend Shared ReadOnly Property PinchedVentTube As Bitmap
            Get
                Return DirectCast(Resources.ResourceManager.GetObject("PinchedVentTube", Resources.resourceCulture), Bitmap)
            End Get
        End Property

        Friend Shared ReadOnly Property VerySmallVentDeltaP As Bitmap
            Get
                Return DirectCast(Resources.ResourceManager.GetObject("VerySmallVentDeltaP", Resources.resourceCulture), Bitmap)
            End Get
        End Property

        Friend Shared ReadOnly Property warning_sm As Bitmap
            Get
                Return DirectCast(Resources.ResourceManager.GetObject("warning_sm", Resources.resourceCulture), Bitmap)
            End Get
        End Property


        ' Fields
        Private Shared resourceMan As ResourceManager
        Private Shared resourceCulture As CultureInfo
    End Class
End Namespace

